synapse/lib/layer.py

```diff
@@ -1101,11 +1102,11 @@ async def clone(self, newdirn):
 
                 relname = os.path.join(relpath, name)
 
-                srcpath = s_common.genpath(root, name)
+                srcpath = pathlib.Path(s_common.genpath(root, name)).resolve()
```

Comment:
genpath is returning a fully resolved absolute path.  Why are we wrapping here?

Reply:
You're right. I need to remove the resolve